DeezyMatch/tests/test_pipeline_one_col_input.py

```diff
@@ -86,7 +86,4 @@ def test_pipeline_one_col_input():
                          pretrained_model_path="./models/finetuned_test003/finetuned_test003.model",
                          pretrained_vocab_path="./models/finetuned_test003/finetuned_test003.vocab",
                          number_test_rows=5)
-        
-    for s in candidates_pd["query"].to_list():
-        assert candidates_pd.loc[candidates_pd["query"] == s]["faiss_distance"].iloc[0][s] == pytest.approx(0.0)
```

Comment:
Is it possible to move this to `DeezyMatch/tests/test_pipeline_ngram.py` where we use only one input?

Reply:
I guess we could keep this test if we used the same one-column dataset (either `dataset-queries.txt` or `dataset-candidates.txt`) for generating both the queries and candidates.

Reply:
The reason that I suggested `DeezyMatch/tests/test_pipeline_ngram.py` is because we are using one file throughout the test, so I think the test should hold in that case. What do you think?

Reply:
Hi @kasra-hosseini, yes, I think it is already there, right?

Reply:
Oh, sorry, I was just looking at the diffs...my bad...yes, all good.